@ABLITZER/plotOntogenyByPI.m

```diff
@@ -1,15 +1,16 @@
 % plot performance versus fish age
 % Here, we use the substraction of PI in training with PI in baseline as
 % the characteristic PI for fish at the same age
+% CAUTION: NOT CORRECT
 function  plotOntogenyByPI(obj,metricType)
```

Comment:
We may need to rewrite this function so that it would generate figures just as Fig. 4 in the manuscript.